src/Tape.hpp

```diff
@@ -51,24 +51,27 @@
 #ifdef _WIN32
 #define TAPE_HOOK(...)
 #else
-#define TAPE_HOOK(...)                                              \
-do if (tapeRecorder.has_value()) {                                    \
-	tapeRecorder->recordApiCall(__func__ __VA_OPT__(, ) __VA_ARGS__); \
-} while (0)
+#define TAPE_HOOK(...)                                                                                                         \
+	do                                                                                                                         \
+		if (tapeRecorder.has_value()) {                                                                                        \
+			tapeRecorder->recordApiCall(__func__ __VA_OPT__(, ) __VA_ARGS__);                                                  \
+		}                                                                                                                      \
+	while (0)
 #endif // _WIN32
 
 #define TAPE_ARRAY(data, count) std::make_pair(data, count)
-#define FWRITE(source, elemSize, elemCount, file)                          \
-do if (fwrite(source, elemSize, elemCount, file) != elemCount) {           \
-    throw RecordError(fmt::format("Failed to write data to binary file")); \
-} while(0)
+#define FWRITE(source, elemSize, elemCount, file)                                                                              \
+	do                                                                                                                         \
+		if (fwrite(source, elemSize, elemCount, file) != elemCount) {                                                          \
+			throw RecordError(fmt::format("Failed to write data to binary file"));                                             \
+		}                                                                                                                      \
+	while (0)
 
 // Type used as a key in TapePlayer object registry
 using TapeAPIObjectID = size_t;
 
-class TapeRecorder
-{
-	YAML::Node yamlRoot; // Represents the whole yaml file
+class TapeRecorder {
+	YAML::Node yamlRoot;      // Represents the whole yaml file
```

Comment:
Bracket moved. It didn't move in other cases. Something's wrong here.

Reply:
Perhaps it's because other are structs, which may be treated differently by clang-format.

Reply:
So do we prefer this bracket to be on a new line?